pandas/core/frame.py

```diff
@@ -5839,7 +5839,7 @@ def unstack(self, level=-1, fill_value=None):
         return unstack(self, level, fill_value)
 
     _shared_docs['melt'] = ("""
-    "Unpivots" a DataFrame from wide format to long format, optionally
+    'Unpivots' a DataFrame from wide format to long format, optionally
```

Comment:
I think we can simply remove the quotes. But in any case, we should fix the validation script to not report cases where the first character is not a letter.

Reply:
So, which fix should I make? Removing the quotes or fixing the validation script? Or both?

Reply:
in this PR remove the quotes, if you want to change the script, that should be im a separate PR